client.go

```diff
@@ -491,6 +482,30 @@ func writeIndex(ctx context.Context, index *ocispec.Index, client *Client, ref s
 	return writeContent(ctx, client.ContentStore(), ocispec.MediaTypeImageIndex, ref, bytes.NewReader(data), content.WithLabels(labels))
 }
 
+// GetLabel gets a label value from namespace store and saves it in 'out' variable.
+// If there is no value, a fallback value will be used instead.
+func (c *Client) GetLabel(ctx context.Context, label string, out *string, fallback string) error {
+	ns, err := namespaces.NamespaceRequired(ctx)
```

Comment:
I think we don't need to check namespace here. For current design, the namespace design doesn't support nested.
everyone can list all namespaces without specific ns.

Reply:
and golang allows us to return multiple variables. 
just my two cents, the return value is more clear than pointer one.

Reply:
> I think we don't need to check namespace here

Right, it's not a check, I use it in `srv.Labels(ctx, ns)`, so we still would have to pass a namespace somehow (either `ctx` or string param). Because most of the time we already have a `ctx` with namespace info, I think `ctx` would be better fit here (otherwise we need to call `namespaces.NamespaceRequired` before `GetLabel` to get `ns` first).

> the return value is more clear than pointer one.

Technically yes, but there is a bit different motivation here. We'll have to return `(error, ok, value)` and the caller we'll have to check for errors first (if we're failed to query labels from store), then check bool `ok` (in order to verify that the value exists in map), assign value or use fallback value. So it'll be a lot more verbose for caller.

Reply:
I'm not sure. I don't really like how the string pointer is used.  I would rather have multiple GetLabel funcs like we have with Namespace/NamespaceRequired

Reply:
We also do not support empty valued labels. So just returning an empty string is adequate to know the label is unset.

Reply:
@crosbymichael @dmcgowan that makes sense. I reworked `GetLabel` to get rid of pointers and return `(string, error)`. So there are 3 cases possible: an error returned (if we couldn't get labels from the store), or nil error and an empty label (if there is no default label), or label itself.